plaidml/bridge/openvino/CMakeLists.txt

```diff
@@ -99,18 +99,20 @@ install(
         ${OUTPUT_ROOT}/bin/libplaidml.so.1
         ${OpenVINO_MAIN_SOURCE_DIR}/${BIN_FOLDER}/${CMAKE_BUILD_TYPE}/lib/libformat_reader.so
         ${IE_MAIN_SOURCE_DIR}/temp/tbb/lib/libtbb.so.2
+        ${IE_MAIN_SOURCE_DIR}/thirdparty/clDNN/common/intel_ocl_icd/6.3/linux/Release/bin/x64/libOpenCL.so.1
```

Comment:
I'm not sure this is a good idea. Maybe we should ensure that libOpenCL is installed on the system?